controller/iteration_blackbox_test.go

```diff
@@ -265,20 +266,47 @@ func (rest *TestIterationREST) TestFailValidationIterationNameStartWith() {
 	assert.Contains(rest.T(), err.Error(), "type.name must match the regexp")
 }
 
-func (rest *TestIterationREST) TestShowIterationOK() {
-
+func (rest *TestIterationREST) TestShow() {
 	// given
-	fxt := tf.NewTestFixture(rest.T(), rest.DB, createSpaceAndRootAreaAndIterations()...)
-	itr := *fxt.Iterations[1]
+	fxt := tf.NewTestFixture(rest.T(), rest.DB,
+		tf.CreateWorkItemEnvironment(),
+		tf.Iterations(2,
+			tf.SetIterationNames("root", "child"),
+			func(fxt *tf.TestFixture, idx int) error {
+				switch idx {
+				case 1:
+					start, err := time.Parse(time.RFC822, "02 Jan 06 15:04 MST")
+					if err != nil {
+						return err
+					}
+					fxt.Iterations[idx].StartAt = &start
+					fxt.Iterations[idx].EndAt = ptr.Time(start.Add(time.Hour * 24 * 365 * 100))
+				}
+				return nil
+			}),
+		tf.Areas(2),
```

Comment:
@kwk I don't think we're using `fxt.Areas` anywhere in this test.

Reply:
Yes, right. It doesn't hurt though.